CRM/Core/Component.php

```diff
@@ -170,10 +167,6 @@ public static function invoke(&$args, $type) {
           if (!empty($comp->info[$name]['formTpl'])) {
             $template->assign('formTpl', $comp->info[$name]['formTpl']);
           }
-          if (!empty($comp->info[$name]['css'])) {
-            $styleSheets = '<style type="text/css">@import url(' . "{$config->resourceBase}css/{$comp->info[$name]['css']});</style>";
-            CRM_Utils_System::addHTMLHead($styleSheet);
-          }
```

Comment:
Even if any components did come with their own css files (they don't), this code never worked because the variable name is misspelled (written to as `$styleSheets` but read as `$styleSheet`) :facepalm: